python/sglang/srt/layers/activation.py

```diff
@@ -64,6 +69,13 @@ def forward_cuda(self, x: torch.Tensor) -> torch.Tensor:
         return out
 
 
+if is_hip():
```

Comment:
1. move this to the end of this file
2. do you need to skip the import of flashinfer at the beginning of this file?

Reply:
@merrymercy just done 1. it is fine to keep as it is for 2.